core/src/test/scala/org/apache/spark/h2o/backends/SharedBackendUtilsTestSuite.scala

```diff
@@ -0,0 +1,83 @@
+package org.apache.spark.h2o.backends
+
+import org.scalatest.{FunSuite, Matchers}
+
+class SharedBackendUtilsTestSuite extends FunSuite with Matchers {
+
+  val expectedHttpHeaderArgs = Seq(
+    "-add_http_header", "X-Frame-Options", "deny",
```

Comment:
it might be misleading if used by an external person as an example - these headers are already built-in H2O

Reply:
These headers are used only for testing the parser. But no problem, I can uses different ones ;-)

Reply:
yeah, I know - it is not an actual issue - it is just not the best example since it is redundant in actual deployment

your call - if you want to change it or not